components/src/core/InfoListItem/InfoListItemStyledComponents.tsx

```diff
@@ -166,11 +166,13 @@ export const Info = styled(Typography, {
     name: 'info-list-item',
     slot: 'info',
     shouldForwardProp: (prop) => prop !== 'fontColor',
-})<Pick<InfoListItemProps & TypographyProps, 'fontColor' | 'noWrap'>>(({ fontColor, theme }) => ({
-    fontWeight: 400,
-    lineHeight: 1.3,
-    color: fontColor || (theme.palette.mode === 'dark' ? theme.palette.text.secondary : 'inherit'),
-}));
+})<Pick<InfoListItemProps & TypographyProps & BoxProps, 'fontColor' | 'component' | 'noWrap'>>(
```

Comment:
Why do you need BoxProps in here?

Reply:
I don't understand... why does `info` have to do with `BoxProps`? And would you please elaborate what you mean by "we were not passing before"?